ui/.eslintrc.js

```diff
@@ -27,5 +28,9 @@ module.exports = {
         "ts-ignore": "allow-with-description",
       },
     ],
```

Comment:
Is it worth adding the additional config for the hooks plugin?
`   "react-hooks/rules-of-hooks": "error", // Checks rules of Hooks `
`   "react-hooks/exhaustive-deps": "warn" // Checks effect dependencies`

Reply:
By extending the recommended eslint config, those rules (and more) should be enabled by default. We can update em later if we find some annoying.